packages/zen/src/plugins/WebpackPlugin.ts

```diff
@@ -71,8 +71,8 @@ const createPlugins = (builder: Builder, zen: Zen) => {
   }
 
   if (builder.require.probe('clean-webpack-plugin')) {
-    const CleanWebpackPlugin = builder.require('clean-webpack-plugin');
-    plugins = plugins.concat(new CleanWebpackPlugin(builder.buildDir));
+    const { CleanWebpackPlugin } = builder.require('clean-webpack-plugin');
+    plugins = plugins.concat(new CleanWebpackPlugin());
```

Comment:
@veeramarni Thanks, why don't you pass `builder.buildDir` into `CleanWebpackPlugin` constructor, as it was done before? Can we support both old and new `clean-webpack-plugin` via code like this:
```
    let CleanWebpackPlugin = builder.require('clean-webpack-plugin');
    CleanWebpackPlugin = CleanWebpackPlugin.CleanWebpackPlugin || CleanWebpackPlugin;
    plugins = plugins.concat(new CleanWebpackPlugin(builder.buildDir));
```

Reply:
Will make that change.